### DISCUSSION TITLE
Ghost-Mode

### DISCUSSION THREAD

**Discussion by Rapheltron** [Dreamlinerm/Netflix-Prime-Auto-Skip#104] [Ideas]
I think it would be useful to have a function that doesn't fast-forward the ads, but just makes them silent and the screen black. My thought is that the streaming service won't find out that you're using this great add-on.

Besides, I have nothing against commercial breaks, only against commercials.

**Comment by Dreamlinerm** [comment]
@Rapheltron  If you are talking about Netflix, then this is already possible if you enable the epilepsy mode in the settings. I think Netflix ads are already muted.

**Comment by Dreamlinerm** [comment]
I did not allways want to make the screen black because it could be confusing if the screen is suddenly black, that's why it's an option

**Comment by Rapheltron** [comment]
I already use the epilepsy mode. I'm more interested in being able to set the speed at which the ads are played.

**Comment by Dreamlinerm** [comment]
@Rapheltron  you can just change the speed with the speed slider and disable the skip ad function. I could also add a setting, but it may cause problems
